src/components/transformation/create/DerivationEditor/index.tsx

```diff
@@ -96,16 +142,28 @@ function DerivationEditor() {
                 </Grid>
             </Grid>
 
-            <Grid
-                item
-                xs={12}
-                sx={{ mt: 2, display: 'flex', justifyContent: 'flex-end' }}
-            >
-                <Button variant="outlined" sx={{ mr: 2 }}>
-                    Proceed to GitPod
-                </Button>
+            <Grid item xs={12} sx={{ mt: 2 }}>
+                <Stack
+                    spacing={2}
+                    direction="row"
+                    sx={{ justifyContent: 'flex-end' }}
+                >
+                    <GitPodButton
+                        postWindowOpen={postWindowOpen}
+                        buttonVariant="outlined"
+                    />
 
-                <Button>Publish</Button>
+                    <EntitySaveButton
+                        callFailed={helpers.callFailed}
+                        taskNames={
```

Comment:
This condition _should_ be able to be removed since the default value of the `catalogName` transform create state property was changed from `null` to an empty string. Will update.